composer.json

```diff
@@ -16,6 +16,7 @@
         }
     ],
     "require":      {
+        "php": ">=5.6",
         "knplabs/gaufrette": "~0.1.7|~0.2|~0.3|~0.4|~0.5|~0.6|~0.7",
```

Comment:
This line should be updated too to require `~0.4` as minimum (see the [release note](https://github.com/KnpLabs/Gaufrette/releases/tag/v0.4.0) where it requires php >=5.6).